--- a/Config.in
+++ b/Config.in
@@ -18,6 +18,10 @@ menu "Global build settings"
 		bool "Cryptographically sign package lists"
 		default y
 
+	config SIGNATURE_CHECK
+		bool "Enable signature checking in opkg"
+		default SIGNED_PACKAGES
+
 endmenu
 
 menu "Advanced configuration options (for developers)"

--- a/Config-build.in
+++ b/Config-build.in
@@ -1064,7 +1064,7 @@
 
 config KERNEL_XDP_SOCKETS
 	bool
-	default n
+	default y
 
 config KERNEL_WIRELESS_EXT
 	bool

